etc/config.xml

```diff
@@ -57,13 +57,17 @@
             </mode>
         </csp>
         <crontab>
-            <yotpo>
```

Comment:
Config.xml is useful to load the default configuration settings for the first time. So this will not be an issue for a customer who installs the module first time.

Regarding existing customers , I understand that we only installed the new extension on 3 beta customers so far and we need to reinstall it after finishing our fixes. As the custom cron configuration is only for 3 fields (orders,catalog,customers), entering the custom values once again there and hitting save button won't be a big problem I believe.

Please let me know your thoughts.

Reply:
I'm aligned. Not a big issue for the 3 beta customers.
Wanted to make sure that:

1. I understand correctly.
2. We communicate well with the customers.